examples/kv_addr_book/include/kv_addr_book.hpp

```diff
@@ -58,8 +56,8 @@ struct person_factory {
       std::string personal_id) {
          return person {
             .account_name = account_name,
-            .first_name = {account_name, first_name},
-            .last_name = {account_name, last_name},
+            .first_name = {first_name, last_name},
```

Comment:
why do you add last_name to first_name?

Reply:
it is how currently the non-unique indexes work.
in order to call a `range()` for `last_name_idx` non-unique index with a fixed value for last name, you have to do it this way, declare the first name as a eosio::non_unique tuple